src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/ExecutableStatementCountCheckTest.java

```diff
@@ -75,6 +75,7 @@ public void testMaxZero() throws Exception {
             "67:5: " + getCheckMessage(MSG_KEY, 2, 0),
             "76:5: " + getCheckMessage(MSG_KEY, 2, 0),
             "79:13: " + getCheckMessage(MSG_KEY, 1, 0),
+            "90:29: " + getCheckMessage(MSG_KEY, 1, 0),
```

Comment:
it is not that easy.
Please add more test cases with lambdas like `a -> a.trim()` where executable statement count should be 1.